qualtran/_infra/bloq.py

```diff
@@ -360,8 +360,9 @@ def bloq_counts(
         Returns:
             A dictionary mapping subbloq to the number of times they appear in the decomposition.
         """
-        _, sigma = self.call_graph(generalizer=generalizer, max_depth=1)
-        return sigma
+        from qualtran.resource_counting import get_bloq_callee_counts
+
+        return dict(get_bloq_callee_counts(self, generalizer=generalizer))
```

Comment:
We should change the return type to a `Mapping` instead of a dictionary so we can also return things like a `Counter`, which is more natural in this case.

Reply:
I thought the general advice is to be specific with your return types. Here, it's actually a dictionary